taxonomies/categories.txt

```diff
@@ -69431,7 +69489,7 @@ hu:Dzsemek, Lekvárok
 it:Confetture, Confettura
 nl:Jams, Confituren
 pl:Dżemy, konfitury, powidła
-pt:Geléias
+pt:Doces
```

Comment:
Hi @moon-rabbitOFF ,
There's an issue with this change:

Errors in the categories taxonomy definition:
ERROR - pt:doces already is a synonym of pt:bombons for entry en:bonbons - pt:doces cannot be mapped to entry en:jams

Reply:
I fixed it